src/content-helper/editor-sidebar/title-suggestions/component-settings.tsx

```diff
@@ -35,17 +38,18 @@ type TitleSuggestionsSettingsProps = {
  * @param {TitleSuggestionsSettingsProps} props The component props.
  */
 export const TitleSuggestionsSettings = ( {
-	tone,
-	persona,
-	onToneChange,
-	onPersonaChange,
 	isLoading,
+	isOpen,
+	onPersonaChange,
+	onSettingChange,
+	onToneChange,
+	persona,
+	tone,
 }: TitleSuggestionsSettingsProps ): JSX.Element => {
-	const [ isSettingActive, setIsSettingActive ] = useState<boolean>( false );
-	const [ isToneSelected, setIsToneSelected ] = useState<boolean>( false );
-	const [ isPersonaSelected, setIsPersonaSelected ] = useState<boolean>( false );
```

Comment:
I've changed the component to always display the tone/persona. When there's no user selection, the defaults will be used as before.